lib/configGenerator/generateAnsibleConfig.js

```diff
@@ -133,11 +133,11 @@ async function generateAnsibleConfig(network, networkName, masternodesCount, see
     const collateralPrivateKey = new PrivateKey(undefined, network);
 
     const blsSignatures = await BlsSignatures();
-    const { PrivateKey: BlsPrivateKey } = blsSignatures;
+    const { PrivateKey: BlsPrivateKey, BasicSchemeMPL } = blsSignatures;
 
     const randomBytes = new Uint8Array(crypto.randomBytes(256));
-    const operatorPrivateKey = BlsPrivateKey.fromBytes(randomBytes, true);
-    const operatorPublicKey = operatorPrivateKey.getPublicKey();
+    const operatorPrivateKey = BlsPrivateKey.from_bytes(randomBytes, true);
```

Comment:
This line fails for me with output `6014136`. I think you need to do `gen_key()` with the random bytes first, then `sk.serialize()` the resulting key as input to this function? I had this error when I tried as well.